src/version.cpp

```diff
@@ -415,10 +424,13 @@ void Exiv2::dumpLibraryInfo(std::ostream& os,const exv_grep_keys_t& keys)
      use_ssh=1;
 #endif
 
-#ifdef EXV_HAVE_REGEX_H
-      have_regex=1;
-#endif
+#define PUSH_PATH(path,libs,paths)  \
```

Comment:
I would prefer to have a static function in this file rather than a MACRO.

Reply:
I agree.  That's a dirty old "C" hack.  Shame on me.

I'll update the PR on Saturday.  I'd like to polish README.md a little and clarify some matters.